cli/scorecard/cmd.go

```diff
@@ -40,6 +41,7 @@ func init() {
 	Cmd.Flags().StringVar(&flags.dirPath, "dir-path", "", "Local directory path for storing inventory (conflicts with --bucket or --stdin)")
 	Cmd.Flags().BoolVar(&flags.stdin, "stdin", false, "Passed Cloud Asset Inventory json string as standard input (conflicts with --dir-path or --bucket)")
 	Cmd.Flags().BoolVar(&flags.refresh, "refresh", false, "Refresh Cloud Asset Inventory export files in GCS bucket. If set, Application Default Credentials must be a service account (Works with --bucket)")
+	Cmd.Flags().BoolVar(&flags.concurrency, "concurrency", false, "Concurrent Violations Review. If set, the CFT application will run the violations review concurrently and may improve the total execution time of the application.")
```

Comment:
I'm wondering if this should be an integer instead to specify the number of workers? That way people can experiment with different worker pool sizes depending on their CPU / data size.

Reply:
changing this to workers optional flag with an int